common/script/libs/statsComputed.js

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash';
+import content from '../content/index';
+import splitWhitespace from '../libs/splitWhitespace';
+const $w = splitWhitespace;
```

Comment:
Probably not necessary since we're removing the place that uses this, but you could have just done: `import $w from '../libs/splitWhitespace';`